src/ngraph_backend_config.cc

```diff
@@ -36,7 +36,12 @@ string BackendConfig::Join(
   } catch (std::out_of_range e1) {
     throw std::out_of_range("Attribute device_config not found");
   }
-  return backend_name_ + ":" + additional_parameters.at("device_config");
+  string backend_name = backend_name_;
+  if (additional_parameters.at("device_config") != "") {
+    backend_name =
+        backend_name + ":" + additional_parameters.at("device_config");
+  }
+  return backend_name;
```

Comment:
I would think the backend should be able to take in the string we were sending without any issue, but seems not.
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/backend.hpp#L56